runtime/themes/everblush.toml

```diff
@@ -0,0 +1,132 @@
+# Author: Isotoxal <isotoxal@proton.me>
+
+"attribute" = { fg = "blue" }
+"boolean" = { fg = "cyan" }
+"character" = { fg = "blue" }
+"comment" = { fg = "comment", italic = true }
+"conditional" = { fg = "red" }
+"constant" = { fg = "cyan" }
+"constant.builtin" = { fg = "blue" }
+"constant.macro" = { fg = "yellow" }
+"constructor" = { fg = "blue" }
+"exception" = { fg = "black-light" }
+"field" = { fg = "red" }
+"float" = { fg = "black-light" }
+"function" = { fg = "red" }
+"function.builtin" = { fg = "cyan-light" }
+"function.macro" = { fg = "green" }
+"include" = { fg = "red-light" }
+"keyword" = { fg = "blue" }
+"keyword.function" = { fg = "blue" }
+"keyword.operator" = { fg = "blue-light" }
+"keyword.return" = { fg = "blue" }
+"label" = { fg = "blue" }
+"method" = { fg = "blue-light" }
+"namespace" = { fg = "red-light" }
+"number" = { fg = "yellow" }
+"operator" = { fg = "white" }
+"parameter" = { fg = "red" }
+"parameter.reference" = { fg = "red-light" }
+"property" = { fg = "red" }
+"punctuation.bracket" = { fg = "white" }
+"punctuation.delimiter" = { fg = "white" }
+"punctuation.special" = { fg = "white" }
+"repeat" = { fg = "yellow-light" }
+"string" = { fg = "green" }
+"string.escape" = { fg = "blue" }
+"string.regex" = { fg = "green" }
+"string.special" = { fg = "blue" }
+"symbol" = { fg = "red" }
+"tag" = { fg = "blue" }
+"tag.attribute" = { fg = "red" }
+"tag.delimiter" = { fg = "white" }
+"text" = { fg = "white" }
+"text.danger" = { fg = "black-light" }
+"text.emphasis" = { fg = "white", italic = true }
+"text.environment.name" = { fg = "yellow" }
+"text.environtment" = { fg = "magenta" }
+"text.literal" = { fg = "green", italic = true }
+"text.math" = { fg = "cyan" }
+"text.note" = { fg = "black-light" }
+"text.reference" = { fg = "cyan" }
+"text.strike" = { fg = "white", strikethrough = true }
+"text.strong" = { fg = "white", bold = true }
+"text.title" = { fg = "yellow", bold = true }
+"text.underline" = { fg = "magenta", underline = true }
+"text.uri" = { fg = "yellow", underline = true }
```

Comment:
Modifiers like strikethrough/bold/underline/italic should be specified in the `modifiers` list instead (https://docs.helix-editor.com/master/themes.html#modifiers):

```toml
"text.strike" = { fg = "white", modifiers = ["strikethrough"] }
```

These should be `markup.*` scopes instead though, for example https://github.com/helix-editor/helix/blob/98a3d46912be7dcc650c54ea417d7f00ab6d05a3/runtime/themes/darcula.toml#L56-L63